src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(health = 100, name) {
+    this.health = health;
+    this.name = name;
+  }
```

Comment:
[CODE LOGIC] In the Animal constructor, you are setting `health` as the first parameter, but when you extend this class in Herbivore and Carnivore, you are passing `name` as the first argument to `super()`. This means that `name` is being assigned to `health` and `health` is being assigned to `name`. To fix this, you should keep the parameters in the same order in both the parent and child classes.